requirements-docker.txt

```diff
@@ -4,4 +4,4 @@ pyOpenSSL==19.1.0
 python-dotenv==0.11.0
 python-vagrant==0.5.15
 j2cli==0.3.10
-
+docker-compose
```

Comment:
add a version

Reply:
OK, I've changed that.

and just for the reference, it was without a specific version in the Dockerfile, so this is why I thought that it could be the latest version

https://github.com/devopsloft/devopsloft/blob/0fcfae315e4786f8e9d30fca885944fb0e626755/Dockerfile#L16

Reply:
I see. We can do some improvements :-)